apps/charterafrica/src/components/Mooc/Mooc.js

```diff
@@ -0,0 +1,100 @@
+import { Section } from "@commons-ui/core";
+import { Figure } from "@commons-ui/next";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import React from "react";
+
+import { neutral } from "@/charterafrica/colors";
+import LineClampedRichTypography from "@/charterafrica/components/LineClampedRichTypography";
+
+const Mooc = React.forwardRef(function Mooc(props, ref) {
+  const { title, link, image, sx } = props;
+
+  if (!title || !link || !image) {
+    return null;
+  }
+
+  return (
+    <Box bgcolor={neutral[900]} ref={ref} sx={sx}>
+      <Section
+        ref={ref}
+        sx={{ px: { xs: 5, sm: 0 }, py: { xs: 5, md: "86px" } }}
+      >
+        <Box
+          display="flex"
+          flexDirection={{ xs: "column-reverse", sm: "row" }}
+          gap={5}
+          justifyContent="center"
+          alignItems="center"
+        >
+          <Box
```

Comment:
Instead of all these `Box`es, you can just use a `Grid`. This is a basic 2 column grid.